build.sh

```diff
@@ -12,10 +12,23 @@
 # Abort script if one command returns a non-zero value
 set -e
 
+for var in "$@"
+do
+    case "${var}" in
+        "-c" | "clean"   ) clean=1;;
+        "-C" | "CLEAN"   ) clean=1 && nobuild=1;;
+        "-i" | "install" ) install=1;;
+        "-t" | "test"    ) test=1;;
+    esac
+done
+
 # Prepare build environment
-if [[ "${1}" == "clean" ]]; then
+if [[ -n "${clean}" ]]; then
+    echo "Cleaning build environment"
     rm -rf cmake/
-    exit 0
+    if [[ -n ${nobuild} ]]; then
+        exit 1
```

Comment:
This reports an error when the script actually did what is was asked to do, is this intentional?

Reply:
Ah no, typo.. was supposed to be `exit 0`
```suggestion
        exit 0
```